src/Models/Nonhydrostatic3DModels/equations_pressure.jl

```diff
@@ -69,17 +69,20 @@ end
     uvw = @. Geometry.Covariant123Vector(uh) +
        Geometry.Covariant123Vector(interp_f2c(w))
     Φ = calculate_gravitational_potential(Y, Ya, params, FT)
-
     e_int = @. ρe_tot / ρ - Φ - norm(uvw)^2 / 2
     q_tot = @. ρq_tot / ρ
+
+    # saturation adjustment
+    maxiter = 1
+    temperature_tol = FT(0.1)
```

Comment:
Why aren't we using the default parameters?

Reply:
From what I remember @tapios has wanted us to use a single Newton iteration.